mlir/test/python/python_test_ops.td

```diff
@@ -227,4 +227,16 @@ def OptionalOperandOp : TestOp<"optional_operand_op"> {
   let results = (outs I32:$result);
 }
 
+def SameVariadicResultSizeOp : TestOp<"same_variadic_result",
+                                      [SameVariadicResultSize]> {
+  let results = (outs Variadic<AnyType>:$variadic1, AnyType:$non_variadic,
```

Comment:
Sorry to be harassing you but can you add tests that permute the non-variadic - so like maybe

non-var, var, var
var, var, non-var

basically I'm too lazy to think really hard about the arithmetic 😅

Reply:
No worries. I’ve added those tests now 😊